css-three-column-layout/styles.css

```diff
@@ -0,0 +1,76 @@
+body {
+  background-color: #1996da;
+  font-family: sans-serif;
+}
+
+.container {
+  margin: auto;
+  width: 75%;
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  flex-direction: column;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+}
+
+.white-text {
+  color: white;
+}
+
+.column-one-third {
+  flex-basis: 28%;
+  padding: 10px;
```

Comment:
The only styling that should be on a `column` class is `width` or `flex-basis`. Make sure the measurement accurately matches the name of the class as well 🤓 . If you need any other additional styling on top of `width` or `flex-basis`, you should make a separate class that you can add to the element in addition to the `column` class.